svg/global_attributes.json

```diff
@@ -902,12 +902,12 @@
           "spec_url": "https://drafts.fxtf.org/css-masking/#clip-property",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
No spec defines a `clip` IDL attribute so https://mdn-bcd-collector.gooborg.com/tests/svg/global_attributes/clip is invalid I think.